batch/batch/server/server.py

```diff
@@ -152,24 +145,35 @@ def _create_pod(self):
         assert self._pod_name is None
         assert self._current_task is not None
 
+        volumes = [
+            kube.client.V1Volume(
+                secret=kube.client.V1SecretVolumeSource(
+                    secret_name=self.userdata['secret_name']),
+                name='gcp-sa-key')]
+        volume_mounts = [
+            kube.client.V1VolumeMount(
+                mount_path='/gcp-sa-key',
```

Comment:
I think it would be useful for us to standardize on a path for the mount. Notebook2 uses /gsa-key.